apps/meteor/tests/e2e/homepage.spec.ts

```diff
@@ -104,11 +172,18 @@ test.describe.serial('homepage', () => {
 			});
 		});
 
-		test.describe('custom body', () => {
+		test.describe('custom body with empty content', () => {
+			test('expect to not show custom content card', async () => {
+				await expect(regularUserPage.locator('[data-qa-id="homepage-custom-content-body"]')).not.toBeVisible();
```

Comment:
same look that: homepage-custom-content-body why not just a css selector so? or even better a role selector?

Reply:
Done!